### ISSUE TITLE
Relative dates from "now" assume zero milliseconds

### ISSUE THREAD

**Issue by smolster** [wanasit/chrono#369]
For example:

```typescript
const inFiveDays = chrono.parseDate('5 days from now');
```

`inFiveDays` always has a millisecond value of 0 (the value is `implied` as 0). However, `parseDate('now')` has an up-to-date millisecond value. This causes strange behavior easily surfaced in tests like so:

```typescript
// mock Date.now()
const now = chrono.parseDate('now');
const inFiveDays = chrono.parseDate('5 days from now');

EXPECTED:
  The difference between now -> inFiveDays is exactly 5 days.

ACTUAL:
  The difference is slightly off from five days, as `inFiveDays` does not have a correct Z
```

 It seems like `inFiveDays` should have a millisecond value equal to the current millisecond value (i.e. calculate `now + 5 * 24 hours`). I'm not familiar enough with the source to know how to address this, so filed this issue.

**Comment by wanasit** [comment]
Thanks. Fixed in 3fd809f.

**Comment by smolster** [comment]
Awesome! Thanks @wanasit!
